mappings/net/minecraft/entity/ai/brain/task/Task.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS ajf net/minecraft/entity/ai/brain/task/Task
+	CLASS ajf$a Status
+	FIELD b timeLimit J
```

Comment:
`timeLimit` seems relative to start time -- iirc this is in relation to time of day

Reply:
I'll name it `endTime`.